tests/status_test.py

```diff
@@ -1,66 +0,0 @@
-from gumbo_client import status
-from datetime import date
-
-
-def test_get_screen_status():
```

Comment:
Fyi, this status logic can be completely removed soon. The existing peddep dashboard still uses it , but the new version of the dashboard does not. Once that's deployed, we can remove all of this from the client (and all this complicated logic around statuses will be handled by Melissa going forward).

Reply:
If that's the case: I'm going to remove this endpoint. :D 

That will allow me to not worry about having it being untested. Also, since I'm deploying to a new URL, I won't break anything when the endpoint goes away.